airflow/models/dagrun.py

```diff
@@ -1065,11 +1065,11 @@ def schedule_tis(self, schedulable_tis: Iterable[TI], session: Session = NEW_SES
         return count
 
     @provide_session
-    def get_log_filename_template(self, *, session: Session = NEW_SESSION) -> str:
```

Comment:
We might want to leave a deprecated `get_log_filename_template` around for backcompat? Only there since 2.3.0, but it's still in releases.

Reply:
I added it back.